app/views/receipt/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Receipt</h1>
+
+<div>Item: <%= @name %></div>
+<div>Price: $<%= @price %> </div>
```

Comment:
I think having `total` instead of price would be better.  A receipt typically has a list of prices and a total.